api/prisma/migrations/17_duplicate_listing_permissions/migration.sql

```diff
@@ -0,0 +1,2 @@
+-- AlterTable
+ALTER TABLE "jurisdictions" ADD COLUMN "duplicate_listing_permissions" "user_role_enum"[];
```

Comment:
nit: missing EOF newline and there is already a migration 17

Reply:
Just addressed this! Good catch 🥫